mariadb/templates/configmaps/seed.sh.yaml.j2

```diff
@@ -0,0 +1,120 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  seed.sh: |+
+    #!/bin/sh
+
+    set -ex
+
+    SLEEP_TIMEOUT=5
+    # we need to count seed instance here.
+    MINIMUM_CLUSTER_SIZE=4
```

Comment:
It do believe that it is very important to keep support for minimal deployments. Stackanetes is really well adapted for developers today and it should remain that way. It is also useful to do workshops/demos/POC, where people only have time for 2 nodes deployment at most.